solidity/nomad-xapps/contracts/bridge/BridgeRouter.sol

```diff
@@ -81,6 +82,9 @@ contract BridgeRouter is Version0, Router {
         uint256 amount
     );
 
+    // ======== Receive =======
+    receive() external payable {}
```

Comment:
hmmm is this necessary, since the BridgeRouter is a proxy that has its own receive fn? i suppose receive falls back to impl so it may indeed also need a receive in impl.

Reply:
i wrote tests for all the combinations of this in my personal solidity playground repo at one point but i can't remember now